### ISSUE TITLE
Repository code files take a long time to load [Android]

### ISSUE THREAD

**Issue by housseindjirdeh** [gitpoint/git-point#122]
Running production builds on an Android device seems to work smoothly except for when I try loading a Repository Code File screen. Not sure why this takes quite a little time for the screen to load.

Would appreciate it if anyone can get a release variant of the Android app on their device and test this out. If I'm not the only one experiencing this - than this is definitely worth digging into 👍

**Comment by patw0929** [comment]
I installed the apk built with master branch codebase, and this is my screen record video:

https://youtu.be/THKqmdTOXr0

Do you mean this screen?
My device is Samsung Note 5 (Android 7.0).

**Comment by housseindjirdeh** [comment]
Yep that's the exact screen I'm referring to (actual code screen).

Do you feel like it's taking a lot longer than it should to load or was it possible that I was just experiencing a little network lag at the time?

**Comment by patw0929** [comment]
Yes, it's really taking longer time to load than other scenes, even if it's only a small LICENSE file. So weird. 🤔

**Comment by housseindjirdeh** [comment]
🤔 🤔 🤔 🤔 🤔 🤔 

Could be syntax highlighting but pull request diffs (which should take longer to load) seem to load much faster 🤔

**Comment by Antoine38660** [comment]
Is it always very slow even with a `release APK`? I can look around the syntax highlighting module

**Comment by machour** [comment]
Just a thought here: If it seems to be slow only on the first page (navigating from repo screen), maybe the problem is that the API call doesn't contain `?ref=master` (and all subsequent calls do).
